eth-contracts/test/delegateManager.test.js

```diff
@@ -401,7 +401,7 @@ contract('DelegateManager', async (accounts) => {
       totalStakedForAccount = await staking.totalStakedFor(stakerAccount)
       spStake = (await serviceProviderFactory.getServiceProviderDetails(stakerAccount)).deployerStake
 
-      await delegateManager.claimRewards({ from: stakerAccount })
+      await delegateManager.claimRewards(stakerAccount, { from: stakerAccount })
```

Comment:
we should have an explicit test for this case. if we just copy one of these claim scenarios and just don't send a from address it should still work right?

Reply:
yep, definitely can add

Reply:
or ideally with a different from: than the stakerAccount

Reply:
yeah i used an entirely different account to confirm the behavior